src/languages/ko-KR.js

```diff
@@ -518,14 +518,14 @@ export default {
     "Show update if available": "사용 가능한 경우에 업데이트 표시",
     "Also check beta release": "베타 릴리즈 확인",
     "Using a Reverse Proxy?": "리버스 프록시를 사용하시나요?",
-    "Check how to config it for WebSocket": "웹소켓에 대한 설정 방법 확인",
+    "Check how to config it for WebSocket": "웹소켓 대한 설정 방법",
     "Steam Game Server": "스팀 게임 서버",
     "Most likely causes:": "원인:",
-    "The resource is no longer available.": "더이상 사용할 수 없어요.",
+    "The resource is no longer available.": "더이상 사용할 수 없어요...",
     "There might be a typing error in the address.": "주소에 오탈자가 있을 수 있어요.",
     "What you can try:": "해결 방법:",
     "Retype the address.": "주소 다시 입력하기",
     "Go back to the previous page.": "이전 페이지로 돌아가기",
-    "Coming Soon": "Coming Soon",
+    "Coming Soon": "Coming Soon...",
```

Comment:
```suggestion
    "Coming Soon": "곧 만나보세요.",
```

If we try to use Korean in all wording, I think this could be used.

Reply:
I don't think it's good because it's too translated.